config.py

```diff
@@ -70,7 +70,7 @@ class LabelEncodingEnum(Enum):
 if MODEL == ModelEnum.BASIC_CNN:
     NB_EPOCHS = 20
 elif MODEL == ModelEnum.PIANOCKTAIL_GRU:
-    NB_EPOCHS = 300
+    NB_EPOCHS = 2000
```

Comment:
I thought it was overfitting at 2000.
Shouldn't we keep this to 300 ?

Reply:
I don’t think it matters much for this merge, the best value will be determined when we have a validation set.